### PR TITLE
feat(stats): Display subseries in tooltip

### PR DESCRIPTION
**Pull request by priscilawebdev**
### Goal

On the stats page, we aim to provide more detailed information about the statistics. Currently, we display the outcomes: Accepted, Filtered, and Dropped, but we should also include specifics about what was filtered and what was dropped.

**Before:**

image

**After:**
image

### Note

Discarded will be included in a follow-up

Part of https://github.com/getsentry/sentry/issues/59320

### CODE REVIEW

**Review by scttcper — APPROVED** [review]

File: static/app/views/organizationStats/usageStatsOrg.tsx [hunk 1]
```diff
@@ -115,7 +117,7 @@ class UsageStatsOrganization<
     return {
       ...queryDatetime,
       interval: getSeriesApiInterval(dataDatetime),
-      groupBy: ['category', 'outcome'],
+      groupBy: ['category', 'outcome', 'reason'],
```
Comment (scttcper) [inline static/app/views/organizationStats/usageStatsOrg.tsx:120]: does this need to be behind a feature flag?

Comment (priscilawebdev) [reply static/app/views/organizationStats/usageStatsOrg.tsx:120]: If I recall correctly, no, but let me check again to confirm.

**Review by vgrozdanic — APPROVED** [review]

**Review by matejminar — APPROVED** [review]
Looks good to me.

Could we do something about this huge tooltip? Perhaps only show non-zero values?
CleanShot 2024-07-04 at 14 45 14@2x

**Comment by priscilawebdev** [comment]
@matejminar sure thing! I created the [issue](https://github.com/getsentry/sentry/issues/73829) and will work on it in a follow-up 😉

**Pull request merged**